src/history.rs

```diff
@@ -79,15 +79,13 @@ pub struct DynamicHistory<'a> {
     /// The number of events seen since the current
     /// probe's logical clock last increased.
     pub(crate) event_count: u32,
-    pub(crate) chunked_report_state: ChunkedReportState,
     /// Invariants:
     ///   * The first clock is always that of the local probe id
     pub(crate) clocks: FixedSliceVec<'a, LogicalClock>,
```

Comment:
This field is now the frontier clocks known as of the read-for-reporting head of the log, right?

Reply:
Seems to be consistently handled thus far.